src/goto-symex/builtin_functions.cpp

```diff
@@ -355,6 +355,61 @@ void goto_symext::symex_printf(const expr2tc &, const expr2tc &rhs)
     cur_state->guard.as_expr(), cur_state->source, fmt.as_string(), args);
 }
 
+void goto_symext::symex_fscanf(
+  const expr2tc&,
+  const std::vector<expr2tc> &op)
+{
+  auto input_stream = op[0]; // stdin?
+  cur_state->rename(input_stream);
+  input_stream->dump();
+
+  /* TODO: Check for Format
+  const irep_idt fmt = get_string_argument(op[1]);
+  std::vector<size_t> alloc_size;
+  */
+
+  for(size_t i = 2; i < op.size(); i++)
```

Comment:
Looping over the number of parameters has wrong semantics. I believe this should always succeed:
```c
int m, n = 42;
fscanf(stdin, "%d", &m, &n);
assert(n == 42);
```

Reply:
PS: The SV-COMP numbers are great, so this (crafted) issue isn't a blocker for me.

Reply:
Yeah, the correct semantics would need to go through the format string. I will add this test as a knownbug though.

Reply:
Thanks @rafaelsamenezes. A comment for the loop would also be nice.